src/Validator/Domain.php

```diff
@@ -45,6 +45,10 @@ public function isValid($value): bool
             return false;
         }
 
+        if (strpos($value, '://') !== false) {
+            return false;
+        }
+
```

Comment:
Hmm maybe it would be better to use the additional filter flag `FILTER_FLAG_HOSTNAME`. See

* https://www.php.net/manual/en/filter.filters.validate.php
* https://www.php.net/manual/en/function.filter-var.php

Reply:
Done - this exposed that we have test cases domains with underscores that are incorrect.